### PR TITLE
[admin]Oppresses Mentors(MentorMute)

### PR DESCRIPTION
**Pull request by redmoogle**
### Intent of your Pull Request

Add Mentormute to mutes

Mutes from

- Mentor PM
- MSAY
- Mentor Help

### Why is this good for the game?

Allows muting mentors or people abusing that, not that it'd help much but /shrug

#### Changelog

:cl:  
rscadd: Add Mentormute
/:cl:

### CODE REVIEW

**Comment by Reed0506** [comment]
oh boy a button to make mentors useless epic

**Comment by Marmio64** [comment]
don't tread on our mhelp rights

**Comment by redmoogle** [comment]
> 
> 
> don't tread on our mhelp rights

Can't stop the badmin train :>

File: yogstation/code/modules/mentor/mentorsay.dm [hunk 1]
```diff
@@ -1,7 +1,9 @@
 /client/proc/cmd_mentor_say(msg as text)
 	set category = "Mentor"
 	set name = "Msay" //Gave this shit a shorter name so you only have to time out "msay" rather than "mentor say" to use it --NeoFite
-
+	if(prefs.muted & MUTE_MENTORHELP)
+		to_chat(src,"<span class='danger'>Error: MSAY: You are unable to use MSAY-s (muted).</span>", confidential = TRUE)
```
Comment (Hopekz) [inline yogstation/code/modules/mentor/mentorsay.dm]:
```suggestion
		to_chat(src,"<span class='danger'>Error: MSAY: You are unable to use MSAY (muted).</span>", confidential = TRUE)
```

File: yogstation/code/modules/mentor/mentorpm.dm [hunk 2]
```diff
@@ -31,6 +31,9 @@
 //Fetching a message if needed. src is the sender and C is the target client
 /client/proc/cmd_mentor_pm(whom, msg, discord_id)
 	var/client/C
+	if(prefs.muted & MUTE_MENTORHELP)
+		to_chat(src,"<span class='danger'>Error: Mentor-PM: You are unable to use Mentor PM-s (muted).</span>", confidential = TRUE)
```
Comment (Hopekz) [inline yogstation/code/modules/mentor/mentorpm.dm]:
```suggestion
		to_chat(src,"<span class='danger'>Error: Mentor-PM: You are unable to use Mentor PM (muted).</span>", confidential = TRUE)
```

**Comment by N3D6** [comment]
why

**Comment by redmoogle** [comment]
> 
> 
> why

Would you like some griffer to spam your mhelps :))))))))), this is just copied adminmute code

**Comment by N3D6** [comment]
downvote relinquished because kek i can't read

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**